WordPress/gutenberg
# Site Editor: Make resizable frame compatible with RTL languages

## What?
This PR ensures that the resize handles in the site editor are positioned correctly in RTL languages.

## How?

I first considered replacing `left` with `inset-inline-start`, which is a logical property, but `framer-motion` doesn't seem to support this property: https://github.com/framer/motion/issues/1989

Therefore, I switched between `left` and `right` depending on the language direction.

## Testing Instructions

- Switch to an RTL locale and access the site editor.
- Press the Tab key to navigate to the resize handle.
- The resize handle should appear between the sidebar navigation and the editor canvas.
- Confirm that you can resize the editor canvas with the left and right keys.
- This should work as usual in LTR languages.

## Screenshots or screencast 

### Before

![before](https://github.com/user-attachments/assets/d9c976b9-0461-46f0-9085-57528969eed4)

### After

![after](https://github.com/user-attachments/assets/b1f987d5-879c-4ab8-93e3-f26015d05a55)
